docs/interfaces/j1708.md

```diff
@@ -0,0 +1,59 @@
+---
+title: J1708
+tags:
+  - J1708
+  - C61
+  - MX-V
+  - CT
+  - CT GL
+---
+
+## Overview
+
+Some of our platforms has J1708 and operating trough uart or spi communication to a co-cpu. supported baudrate is 9600 (faster might be possible but not tested).
+
+To be able to use the j1708 you will need to include /usr/lib/libj1708.so to your application.
+
+For for quick test you can use `j1708-test`.
+```
+root@mx4-ct-29009999:~# j1708-test
```

Comment:
Is this application in $PATH? If not, please provide the absolute path.

Reply:
yes it is for those unit that has j1708